accessors/spanner/spannermetadataaccessor/spanner_metadata_accessor.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package spannermetadataaccessor
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/spanner"
+	spannermetadataclient "github.com/GoogleCloudPlatform/spanner-migration-tool/accessors/spanner/spannermetadataaccessor/clients"
+	"github.com/GoogleCloudPlatform/spanner-migration-tool/common/constants"
+	"google.golang.org/api/iterator"
+)
+
+type SpannerMetadataAccessor interface {
+	// IsSpannerSupportedStatement checks whether the statement from Source database is supported by Spanner or not.
```

Comment:
`IsSpannerSupportedStatement checks whether the statement from Source database is supported by Spanner or not.` -> `IsSpannerSupportedStatement checks if the given statement is supported by Spanner.`

Reply:
ok